lib/Conversion/TorchToTcp/Utils.h

```diff
@@ -22,5 +22,21 @@ Value broadcastShapeInLeadingDims(ConversionPatternRewriter &rewriter,
 Value broadcastInLeadingDimsToMatchShape(ConversionPatternRewriter &rewriter,
                                          Value input, Value target);
 
+Value broadcast0DToNDAndMatchShape(ConversionPatternRewriter &rewriter,
+                                   Value input, Value target);
+
+Value broadcast1DToNDAndMatchShape(ConversionPatternRewriter &rewriter,
+                                   Value input, Value target,
+                                   int64_t axisInOutput);
+
+template <typename T>
+std::optional<Value>
+getConstTensorUtil(PatternRewriter &rewriter, Operation *op, ArrayRef<T> vec,
```

Comment:
Is this just called from `getConstTensor`?  If so, it should be in an `impl` namespace to make it clear that it is a helper.